services/ui-src/src/verbiage/pages/help.ts

```diff
@@ -17,16 +17,5 @@ export default {
       },
     },
   },
-  accordionItems: [
-    {
-      question: "When will the MCPAR online reporting portal be available?",
-      answer:
-        "The MCPAR online reporting portal will be available starting early November 2022.",
-    },
-    {
-      question: "How do I log into my IDM account?",
-      answer:
-        "CMCS staff will be reaching out to states directly to make sure that appropriate state staff are granted access in the very near future so that each state has access in advance of the web-based forms being ready for use. Once the web-based forms are ready, you’ll receive an announcement from CMS.",
-    },
-  ],
+  accordionItems: [],
```

Comment:
Note: When deleting stuff like this, especially if I know I'll be adding some back soon, I sometimes leave some breadcrumbs for myself, like: 

```ts
//  accordion items are in the following format:
//  {
//    question: "",
//    answer: "",
//  }
```